counts.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+from polib import pofile
```

Comment:
```suggestion
from polib import pofile
```
It is odd ruff did not scream about this.

Reply:
pathlib is part of standard library, polib is third-party, I think they are in different [isort groups](https://docs.astral.sh/ruff/rules/#isort-i)